src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -86,6 +121,14 @@ public HubConnection(IConnectionFactory connectionFactory, IHubProtocol protocol
             }
         }
 
+        /// <summary>
+        /// Subscribes a handler that will be raised when a hub method with the specified method and parameters is invoked.
```

Comment:
~~Subscribes~~ **Registers** a handler that will be ~~raised~~ **invoked** when ~~a hub~~ **the** method with the specified ~~method~~  **name** ~~and parameters~~ is invoked.

(the "and parameters" makes it seem like we support overloading, we don't ;))

Reply:
hmm, I think the terminology for call backs being called is they get raised

Reply:
For events I'd agree since the terminology has been around "raising" events (ever since ye olde `RaiseEvent`). Not so sure that's the case when defining "methods" which is really what this is doing. I don't really mind either way though, happy to just let it be `raised` here :)